hms-app/src/pages/YourIdeas.tsx

```diff
@@ -100,7 +104,7 @@ export default function YourIdeas() {
               </div>
             )}
             {!isParticipant() && (
-              <div>you haven't participated in this hackathon</div>
+              <div>you haven&apost participated in this hackathon</div>
```

Comment:
?

Reply:
&apos is the code for `'` in html https://www.educative.io/answers/how-to-create-the-apostrophe-symbol-in-html

Reply:
hm
<img width="1179" alt="Bildschirmfoto 2022-07-22 um 10 35 46" src="https://user-images.githubusercontent.com/59581396/180399238-33bdee46-7755-4207-8ea7-0ccdfccf0d26.png">
n

Reply:
there was a typo: a semicolon missing.
Try again